html-block-and-inline/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+  <html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Document</title>
+    </head>
+    <body>
+
+      <h1>Blue Americana Snuggie&reg; Blanket</h1>
+      <h2>by SnuggieStore</h2>
+      <p>
```

Comment:
While this does work, this should be done using multiple p tags rather than a single tag broken up with break tags. Please update!

Reply:
I took a guess on which way to go. I will update ASAP!